integration/setup/catalog.go

```diff
@@ -0,0 +1,138 @@
+
+	"github.com/giantswarm/azure-operator/integration/env"
+	"github.com/giantswarm/azure-operator/integration/key"
+	"github.com/giantswarm/azure-operator/pkg/project"
+)
+
+const (
+	CatalogStorageURL     = "https://giantswarm.github.io/control-plane-catalog/"
+	TestCatalogStorageURL = "https://giantswarm.github.io/control-plane-test-catalog/"
+)
+
+var (
+	latestOperatorRelease string
+)
+
+func GetLatestOperatorRelease() string {
+	return latestOperatorRelease
+}
+
+func init() {
+	fmt.Printf("calculating latest %#q release\n", project.Name())
+
+	var err error
+	latestOperatorRelease, err = appcatalog.GetLatestVersion(context.Background(), CatalogStorageURL, project.Name())
+	if err != nil {
+		panic(fmt.Sprintln("cannot calculate latest operator release from app catalog"))
+	}
+
+	fmt.Printf("latest %#q release is %#q\n", project.Name(), latestOperatorRelease)
+}
+
+func pullLatestReleaseChartPackage(ctx context.Context, config Config, chartName string) (string, error) {
+	var err error
+
+	var latestRelease string
+	{
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("calculating latest %s release version", chartName))
+		latestRelease, err = appcatalog.GetLatestVersion(ctx, CatalogStorageURL, chartName)
+		if err != nil {
+			return "", microerror.Mask(err)
+		}
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("latest %s release is %s", chartName, latestRelease))
+	}
+
+	var latestReleaseChartPackagePath string
+	{
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("getting tarball URL for latest %s release", chartName))
+		latestReleaseTarballURL, err := appcatalog.NewTarballURL(CatalogStorageURL, chartName, latestRelease)
+		if err != nil {
+			return "", microerror.Mask(err)
+		}
+
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("tarball URL for latest %s release is %s", chartName, latestReleaseTarballURL))
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("pulling tarball for latest %s release", chartName))
+		latestReleaseChartPackagePath, err = config.HelmClient.PullChartTarball(ctx, latestReleaseTarballURL)
+		if err != nil {
+			return "", microerror.Mask(err)
+		}
+
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("tarball path for latest %s release is %s", chartName, latestReleaseChartPackagePath))
+	}
+
+	return latestReleaseChartPackagePath, err
+}
+
+func pullChartPackageUnderTest(ctx context.Context, config Config) (string, error) {
+	config.Logger.LogCtx(ctx, "level", "debug", "message", "getting tarball URL for azure-operator tested version")
+	operatorTarballURL, err := appcatalog.NewTarballURL(TestCatalogStorageURL, project.Name(), fmt.Sprintf("%s-%s", latestOperatorRelease, env.CircleSHA()))
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("tarball URL for azure-operator tested version is %#q", operatorTarballURL))
```

Comment:
For better copy/pasting:

```suggestion
	config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("tarball URL for %#q tested version is %#q", project.Name(), operatorTarballURL))
```